packages/twenty-front/src/modules/activities/emails/right-drawer/hooks/useRightDrawerEmailThread.ts

```diff
@@ -50,6 +50,32 @@ export const useRightDrawerEmailThread = () => {
     skip: !viewableRecordId,
   });
 
+  const lastMessage = messages[messages.length - 1];
+  const lastParticipant =
+    lastMessage?.messageParticipants[
+      lastMessage.messageParticipants.length - 1
+    ];
+  const lastMessageId = lastParticipant?.messageId;
+
+  const {
+    records: messageThreadExternalIdsData,
+    loading: messageThreadExternalIdsLoading,
+    error: messageThreadExternalIdsError,
+  } = useFindManyRecords<MessageThreadExternalId>({
```

Comment:
I dont know how to fetch just one record based on `messageId` here.
in `useFindOneRecord` it asks for the primary id as `objectRecordId` which we dont have in this case.
if we could get `messageThreadExternalId` in the EmailThreadMessage useFindManyRecords call we could save this call. But I could not figure out how :(